app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="center jumbotron">
+	<h1>Welcome to the Sample App</h1>
+
+	<h2>
+		This is the home page for the
+		<a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+		sample application.
+	</h2>
+
+	<%= link_to "Sign up now!", signup_path, class: "btn btn-lg btn-primary" %>
+
+</div>
+
+<%= link_to image_tag("rails.png", alt: "Rails logo"),
+'http://rubyonrails.org/' %>
+<%#= link_to image_tag("kitten.jpg", alt: "Kitten image"),
+            'http://rubyonrails.org/' %>
```

Comment:
Same comment for this file like as the previous comment and remove the comments.